language-support/java/codegen/src/it/java-1.3/com.digitalasset/MapTest.java

```diff
@@ -0,0 +1,269 @@
+                                                                                                ).build())
+                                                                                        .addEntries(ValueOuterClass.Map.Entry.newBuilder()
+                                                                                                .setKey(ValueOuterClass.Value.newBuilder().setText("key2"))
+                                                                                                .setValue(ValueOuterClass.Value.newBuilder()
+                                                                                                        .setRecord(ValueOuterClass.Record.newBuilder()
+                                                                                                                .addFields(ValueOuterClass.RecordField.newBuilder()
+                                                                                                                        .setLabel("value")
+                                                                                                                        .setValue(ValueOuterClass.Value.newBuilder()
+                                                                                                                                .setInt64(2L)
+                                                                                                                                .build())
+                                                                                                                        .build())
+                                                                                                                .build())
+                                                                                                        .build()
+                                                                                                ).build())
+                                                                                )
+                                                                                .build())
+                                                                        .build())
+                                                                .build())
+                                                        .build())
+                                                .build()
+                                        )
+                                        .addEntries(ValueOuterClass.Map.Entry.newBuilder()
+                                                .setKey(ValueOuterClass.Value.newBuilder()
+                                                        .setText("outerkey1"))
+                                                .setValue(ValueOuterClass.Value.newBuilder()
+                                                        .setRecord(ValueOuterClass.Record.newBuilder()
```

Comment:
Same as above, the value of the outer map should be another `Map` and not a `Record`.